packages/mds-agency/request-handlers.ts

```diff
@@ -19,6 +19,7 @@ import {
   UUID
 } from '@mds-core/mds-types'
 import urls from 'url'
+import client from '@mds-core/mds-web-sockets'
```

Comment:
Perhaps `import socket from '@mds-core/mds-web-sockets'` ... `socket.writeEvent` is more narrative.